pkgs/development/tools/extism-cli/default.nix

```diff
@@ -1,31 +1,22 @@
-{ lib, stdenvNoCC, fetchFromGitHub, python3, makeBinaryWrapper }:
+{
+  lib
+, buildGoModule
+, fetchFromGitHub
+}:
 
-stdenvNoCC.mkDerivation rec {
+buildGoModule rec {
   pname = "extism-cli";
-  version = "0.1.0";
-
+  version = "0.3.8";
   src = fetchFromGitHub {
     owner = "extism";
     repo = "cli";
-    rev = "97935786166e82154266b82410028482800e6061";
-    sha256 = "sha256-LRzXuZQt5h3exw43UXUwLVIhveYVFw/SQ2YtHI9ZnWc=";
-  };
-
-  buildInputs = [ python3 ];
-  nativeBuildInputs = [ makeBinaryWrapper ];
-
-  installPhase = ''
-    runHook preInstall
-
-    install -D -m 755 ./extism_cli/__init__.py "$out/bin/extism"
+    rev = "refs/tags/v${version}";
+    hash = "sha256-StMipPMLSQzrhWv0yoKkNiuHMRW7QIhmVZ/M27WDWrM=";
+  } + /extism;
```

Comment:
you can use `modRoot = "./extism";` instead

Reply:
Ah nice, thank you!